src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -276,6 +292,9 @@ object DepositDir {
       addProperty("state.label", depositInfo.state.toString)
       addProperty("state.description", depositInfo.stateDescription)
       addProperty("depositor.userId", user)
+      addProperty("curation.required", "yes")
```

Comment:
not sure (1) if you know this and (2) if this is necessary here, but there is a `BooleanUtils.toStringYesNo` method for the `PropertiesConfiguration` from Apache Commons.